examples/data_science/browse.py

```diff
@@ -0,0 +1,19 @@
+import os
```

Comment:
Maybe rename the file from `browse.py` to `tape_browser.py` to be in line with the similar scripts in the gaia and workarena examples.